python/lsst/ts/mtrotator/rotator_csc.py

```diff
@@ -354,6 +369,56 @@ async def do_configureVelocity(self, data: salobj.BaseMsgType) -> None:
             )
         await self.run_command(code=enums.CommandCode.CONFIG_VEL, param1=data.vlimit)
 
+    async def _do_configureEmergencyAcceleration(
+        self, data: salobj.BaseMsgType
+    ) -> None:
+        """Configure the emergency acceleration limit.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+            Data of the SAL message.
+
+        Raises
+        ------
+        `salobj.ExpectedError`
+            When the value is 0 or negative.
+        """
+
+        self.assert_enabled_substate(EnabledSubstate.STATIONARY)
+
+        alimit = data.alimit
+        if alimit <= 0.0:
+            raise salobj.ExpectedError(f"Emergency alimit={alimit} must be >= 0")
+
+        await self.run_command(
+            code=enums.CommandCode.CONFIG_ACCEL_EMERGENCY, param1=alimit
+        )
+
+    async def _do_configureEmergencyJerk(self, data: salobj.BaseMsgType) -> None:
+        """Configure the emergency jerk limit.
+
+        Parameters
+        ----------
+        data : `salobj.BaseMsgType`
+            Data of the SAL message.
+
+        Raises
+        ------
+        `salobj.ExpectedError`
+            When the value is 0 or negative.
+        """
+
+        self.assert_enabled_substate(EnabledSubstate.STATIONARY)
+
+        jlimit = data.jlimit
+        if jlimit <= 0.0:
+            raise salobj.ExpectedError(f"Emergency jlimit={jlimit} must be >= 0")
```

Comment:
`jlimit={jlimit}` can be written as `{jlimit=}`

Reply:
Updated in the commit: [3ed04af](https://github.com/lsst-ts/ts_mtrotator/pull/86/commits/3ed04af203aa5ccebcfea3464dfa1b4652aea1f6). Thanks!